app/views/defra-id/register-your-interest/registration-complete.njk

```diff
@@ -0,0 +1,21 @@
+{% extends './layouts/layout.njk' %}
+
+{% block pageTitle %}{{serviceName}}{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {{ govukPanel({
+        titleText: "Registration complete - DRAFT PAGE"
```

Comment:
we could replace the placeholder content with https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/170010/ and form the prototype  https://fcp-ahwr-prototype.herokuapp.com/v10/registration/confirmation